apps/server/src/schema/feedback/feedback-types.ts

```diff
@@ -0,0 +1,18 @@
+import { builder } from '../builder';
+
+export const FeedbackTypeEnum = builder.enumType('FeedbackType', {
+  values: {
+    BUG_REPORT: { description: 'Report a bug' },
+    FEATURE_SUGGESTION: { description: 'Suggest a feature' },
+    OTHER: { description: 'Other feedback' },
+  },
+});
+
+export const FeedbackDto = builder.simpleObject('Feedback', {
```

Comment:
```suggestion
export const FeedbackDto = builder.prismaObject('Feedback', {
```
We always prioritze prisma object when possible. Look around to see its syntax